packages/story-editor/src/components/canvas/singleSelectionMoveable/index.js

```diff
@@ -154,6 +163,10 @@ function SingleSelectionMoveable({
    */
   const resetMoveable = useBatchingCallback(
     (target) => {
+      if (!isMounted.current) {
```

Comment:
Same here, I wonder if checking for one of the refs here wouldn't work, e.g. `moveable.current`